.github/workflows/fmt-and-lint.yml

```diff
@@ -7,34 +7,38 @@ concurrency:
 
 jobs:
   fmt-and-lint:
+    name: Style Checks
```

Comment:
this title isn't correct, we lint the code from many aspects which isn't all about styles

```suggestion
    name: X86 Format and Lint Check
```

Reply:
I think we might wanna omit and `X86` part tho. and also have wasm with the same name.
I think this makes the workflow looks more organized:
![image](https://github.com/KomodoPlatform/komodo-defi-framework/assets/56920956/8a13576f-ac4a-4965-ab59-e256b58acc31)